src/pages/GetStarted/GetStartedPlacesMock.js

```diff
@@ -0,0 +1,533 @@
+export default {
```

Comment:
This file is the result of the countryFacets.graphl query. I was just using it to stay in Storybook while working. It can be deleted once you're running the actual query.